lib/virt_feature_test_base.pm

```diff
@@ -160,7 +160,12 @@ sub post_fail_hook {
     #(caller(0))[3] can help pass calling subroutine name into called subroutine
     $self->junit_log_provision((caller(0))[3]) if get_var("VIRT_AUTOTEST");
 
-    virt_utils::collect_host_and_guest_logs;
+    if (get_var('ENABLE_VIR_NET')) {
+        virt_utils::collect_host_and_guest_logs('', '', '/tmp/wickded');
+    }
+    else {
+        virt_utils::collect_host_and_guest_logs;
+    }
     upload_logs("/var/log/clean_up_virt_logs.log");
     save_screenshot;
     upload_logs("/var/log/guest_console_monitor.log");
```

Comment:
Please make sure supportconfig does not collect such logs even with "wicked debug logs = enabled". So there is a chance that your further effort can be saved.

Reply:
Great point. As wicked developer's mentioned, supportconfig are normal logs -- they may show some error or not. What they show is the config and result at the end (except supportconfig has been called after some manual tweaks/fix attempts, ...). 
So, this PR will collect and upload the related wicked logs from guest system to help wicked developer to debug wicked problem.